scripts/postinstall.sh

```diff
@@ -1,3 +1,28 @@
 #!/bin/bash
 
-cp node_modules/wikipedia-preview/dist/wikipedia-preview.production.js assets/js
+mkdir -p assets/images
+cp wikipedia-preview/images/* assets/images
+
+# rewrite the replace method in a better format
+cd wikipedia-preview
+git reset --hard
+npm i
+sed -i "s/inline-svg( '..\/images/url( \/wp-content\/plugins\/wikipediapreview-wordpress\/assets\/images/g" style/*.less
```

Comment:
This is very hacky. If it works well it can be ok for the first release but we should do something like `IMAGE_URL=/wp-content/... EMBED_CSS=false npm run build` to make sure the bundle is generated correctly from within Wikipedia Preview

Reply:
yes, the css part is all hacky. It would be great to have this and require another update from wikipedia-preview itself.